.gitignore

```diff
@@ -5,3 +5,4 @@ pyanalyze/*.c
 build/
 *.code-workspace
 .vscode/
+.venv
```

Comment:
```suggestion
.venv
```

Reply:
I would like this to end in a newline but looks like I got the suggestion wrong.